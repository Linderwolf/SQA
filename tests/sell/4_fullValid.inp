login
full1
sell
Tetris
3.99