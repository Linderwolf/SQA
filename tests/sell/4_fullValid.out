Welcome! :
Please enter your username:
Username accepted! How may we help you today? :
Creating new listing.. What's the game's name? :
How much will the game cost? :
Saving to transaction file.. Success! Game has been added to the system.